sklearn/feature_extraction/hashing.py

```diff
@@ -66,13 +68,18 @@ class FeatureHasher(BaseEstimator, TransformerMixin):
         When True, an alternating sign is added to the features as to
         approximately conserve the inner product in the hashed space even for
         small n_features. This approach is similar to sparse random projection.
+    random_state : int, RandomState instance or None, optional (default=None)
+        If int, random_state is the seed used by the hashing function;
+        If RandomState instance, the seed is the RandomState instance seed;
+        If None, the seed is generated by the RandomState instance used
+        by `np.random`.
```

Comment:
Using the latest docstring formatting convention https://github.com/scikit-learn/scikit-learn/pull/12356, this could be,
```
    random_state : {int, np.random.RandomStateInstance, None}, default=None
       Pseudo-random number generator that defines the see of the hashing
       function. See :term:`random_state`.
````